azure-pipelines.yml

```diff
@@ -14,139 +14,48 @@ pr:
     - '*'
 
 variables:
-  - name: is-runtime-release
-    value: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/Runtime-v')]
-  - name: is-sdk-release
-    value: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/SDK-v')]
-  - name: Codeql.Enabled
-    value: true
+- name: Codeql.Enabled
+  value: true
+
+parameters:
+- name: pools
+  type: object
+  default:
+  - name: NetCore-Public
+    vmImage: ubuntu-latest # The public PR pool schema is different from the 1ES one. We use 'vmImage' which is a default azure pipeline schema.
+    os: linux
+  - name: NetCore-Public
+    vmImage: macOS-latest # 1ES uses 'image' but defining an image this way using our public pools does not work as they aren't configured this way.
+    os: macOS
```

Comment:
I'd just use the Microsoft-hosted one with certainty. Therefore, update `name` to be `Azure Pipelines`.

Reply:
The name does not actually matter at all. But this is probably the most theoretically accurate. Except it might be misleading, maybe do something else like `Unused Pool Name`

Reply:
Itd be better to just get rid of the name but I dont know if we can actually do that or not.

Reply:
Ok based on the docs Name is required. Super awkward that Name becomes the name of the pool using 1es format because that is not the intended use of Name in a non 1es format yaml file. The name is meant to be the name of a paramter, not the pool name, but since we use two templates with different meanings for the same thing, it's going to be weird either way. We could call it 'Fake 1ES Pool Object with Vm Image Windows'